NetKet/Utils/stopwatch.hpp

```diff
@@ -0,0 +1,33 @@
+#ifndef NETKET_STOPWATCH_HPP
+#define NETKET_STOPWATCH_HPP
+
+#include <chrono>
+
+namespace netket {
+
+class Stopwatch
```

Comment:
Is this class actually used anywhere?

Reply:
Yes, in the unit tests.

Reply:
Then shouldn't this header be located somewhere in `Test/?